notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -346,26 +346,6 @@
         "! gsutil mb -l $REGION -p $PROJECT_ID $BUCKET_URI"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #56.    def convert_dataframe_to_dataset(
</pre><p>I'd remove this function and just inline this code. Looks like you only call the function once so n need to have it as a function. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>